src/v/redpanda/admin_server.cc

```diff
@@ -5445,3 +5449,34 @@ admin_server::sampled_memory_profile_handler(
     co_return co_await ss::make_ready_future<ss::json::json_return_type>(
       std::move(resp));
 }
+
+void admin_server::register_wasm_transform_routes() {
+    register_route_raw_async<user>(
+      ss::httpd::transform_json::deploy_transform,
+      [this](
+        std::unique_ptr<ss::http::request> req,
+        std::unique_ptr<ss::http::reply> rep) {
+          return deploy_transform(std::move(req), std::move(rep));
+      });
+    register_route<user>(
+      ss::httpd::transform_json::list_transforms,
+      [this](auto req) { return list_transforms(std::move(req)); });
+    register_route<user>(
```

Comment:
What is the reasoning to allow regular users to `POST /v1/transform/delete`? Should this be superuser only?

Reply:
Same as deploy.